specifications.yml

```diff
@@ -339,6 +359,49 @@ components:
           schema:
             $ref: "#/components/schemas/customerApplication"
 
+    takeDecisionOnApplication:
+      required: true
+      description: Decision Data
+      content:
+        application/json:
+          schema:
+            oneOf:
+              - $ref: "#/components/schemas/BodyForCustomerApplication"
+              - $ref: "#/components/schemas/BodyForFinancingApplication"
+          examples:
+            "Refer Application":
+              value:
+                decision: "referred"
+            "Deny Application":
+              value:
+                decision: "denied"
+                decisionReason: "Reason for denied"
+            "Grant Financing Application":
+              value:
+                decision: "granted"
+                decisionReason: "Reason for granted"
```

Comment:
This is not needed :) If the application is granted, you don't really need to provide a reason